lib/valueflow.cpp

```diff
@@ -3214,6 +3217,12 @@ struct ExpressionAnalyzer : SingleValueFlowAnalyzer {
     bool isVariable() const override {
         return expr->varId() > 0;
     }
+
+    Action isAliasModified(const Token* tok, int indirect) const override {
+        if (value.isSymbolicValue() && tok->astParent() && tok->astParent()->isIncDecOp() && Token::Match(value.tokvalue->astParent(), "%cop%"))
+            indirect = 0;
```

Comment:
Why is this added here?

Reply:
It seems like we should just do:

```cpp
if (value.isSymbolicValue())
    indirect = 0;
```

The check for incrementing will be handled automatically by `isVariableChanged`.

Reply:
Wait, it might need to be:

```cpp
if (value.isSymbolicValue() && tok->exprId() == value.tokvalue->exprId())
    indirect = 0;
```

Since we are checking other types of aliases as well.